compiler-api/src/main/java/com/squareup/anvil/compiler/api/AnvilCompilationException.kt

```diff
@@ -0,0 +1,96 @@
+package com.squareup.anvil.compiler.api
+
+import org.jetbrains.kotlin.codegen.CompilationException
+import org.jetbrains.kotlin.com.intellij.psi.PsiElement
+import org.jetbrains.kotlin.com.intellij.psi.PsiNameIdentifierOwner
+import org.jetbrains.kotlin.descriptors.ClassDescriptor
+import org.jetbrains.kotlin.descriptors.annotations.AnnotationDescriptor
+import org.jetbrains.kotlin.ir.IrElement
+import org.jetbrains.kotlin.ir.declarations.IrClass
+import org.jetbrains.kotlin.ir.symbols.IrSymbol
+import org.jetbrains.kotlin.ir.util.render
+import org.jetbrains.kotlin.js.resolve.diagnostics.findPsi
+import org.jetbrains.kotlin.resolve.source.KotlinSourceElement
+import org.jetbrains.kotlin.resolve.source.getPsi
+import org.jetbrains.kotlin.util.getExceptionMessage
+
+/**
+ * `AnvilCompilationException` is thrown when Anvil specific can't be processed or an error
```

Comment:
This reads strangely. What does "when Anvil specific can't be processed" mean?

Reply:
I forgot to type the word "code", "when Anvil specific code can't be processed".